test/socket.cpp

```diff
@@ -542,7 +542,7 @@ BOOST_AUTO_TEST_CASE(keepalive_timeout)
         , [&](error_code error) {
             BOOST_VERIFY(!error);
             tested_client = true;
-            client_socket.close();
+            //client_socket.close();  // NOTE TEMPORARY WORKAROUND PENDING MAID-885
```

Comment:
I'm not very happy about this as it makes the test irrelevant. I think its better for a test to indicate there is a problem rather than give false positives.